src/vs/workbench/parts/files/electron-browser/views/explorerViewer.ts

```diff
@@ -300,6 +300,7 @@ export class FileRenderer implements IRenderer {
 
 		const done = once((commit: boolean, blur: boolean) => {
 			tree.clearHighlight();
+			label.element.style.display = 'none';
```

Comment:
line 249: templateData.label.element.style.display = 'flex';
leads to two visible labels.
By setting the old label to display = 'none' this can be avoided.